### PR TITLE
648 front end changes for duplicating sections

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Front end changes made for the functionality of duplicating a section. Added duplicate button on section page and functionality behind it

### How to review 
1. Run tests to make sure they pass
1. Make sure you can duplicate a section with questions, answers and validations.

### Dependencies
- [x] - https://github.com/ONSdigital/eq-author-api/pull/150

### CODE REVIEW

**Comment by adammahmood** [comment]
@rmccar Does this have a dependency on an API PR?

**Comment by rmccar** [comment]
@adammahmood Yes it does, theres a card on trello for the back end that hasn't been done yet. Currently it just returns an error from the API saying that the functionality hasn't been implemented yet

**Comment by adammahmood** [comment]
I think it's best to tag this PR as don't merge and has dependency until the backend work is complete.

**Comment by rmccar** [comment]
Yeah good idea

File: src/components/SectionRoute/index.js [hunk 1]
```diff
@@ -83,6 +85,19 @@ export class UnwrappedSectionRoute extends React.Component {
     this.props.onAddPage(sectionId, 0);
   };
 
+  handleDuplicateSection = e => {
+    e.preventDefault();
```
[the remark this answers is not in the document]
Comment (rmccar) [reply src/components/SectionRoute/index.js]: To be honest I was just copying the code for duplicate page, so I think this is used there too

[the remark this answers is not in the document]
Comment (tgandrews) [reply src/components/SectionRoute/index.js]: I think we should work out why its there and as its a bit of a smell.

**Pull request merged**